back-end/branch/apps.py

```diff
@@ -0,0 +1,6 @@
+from django.apps import AppConfig
+
+
```

Comment:
Why is a separate app for branch required. I think we do not need to create such fine abstractions. We can go ahead with an enum of branches and whenever, adding a new branch is required it will be a fix in the backend and a migration is required.